Zend/zend.c

```diff
@@ -103,6 +136,7 @@ static ZEND_INI_MH(OnUpdateScriptEncoding) /* {{{ */
 ZEND_INI_BEGIN()
 	ZEND_INI_ENTRY("error_reporting",				NULL,		ZEND_INI_ALL,		OnUpdateErrorReporting)
 	STD_ZEND_INI_BOOLEAN("zend.enable_gc",				"1",	ZEND_INI_ALL,		OnUpdateGCEnabled,      gc_enabled,     zend_gc_globals,        gc_globals)
+	ZEND_INI_ENTRY("zend.script_extensions",		".php .phar",		ZEND_INI_ALL,		OnUpdateScriptExtensions)
```

Comment:
requiring . means if somebody puts inc there (instead of .inc) they are still vulnerable.